src/pages/Filing/FilingApp/FileDetailsValidation.tsx

```diff
@@ -26,7 +26,7 @@ function FileDetailsValidation({
   const validationStatusMessage: string =
     hasWarnings &&
     dataGetSubmissionLatest.state === FileSubmissionState.SUBMISSION_ACCEPTED
-      ? 'Your file contains no errors and warnings have been verified'
+      ? 'Your file contained no errors and warnings have been verified. Navigate through the validation results pages to continue to the required next step.'
```

Comment:
Change to: 
"Your file contains no errors and warnings have been verified. Navigate through the validation results pages to continue."